src/boot/loader.tsx

```diff
@@ -105,6 +117,78 @@ export const Loader = (): React.JSX.Element => {
 		return () => {
 			unloadAllApps();
 		};
-	}, []);
+	}, [getSessionInfo]);
+
+	useEffect(() => {
+		const expirationTimeouts: NodeJS.Timeout[] = [];
+		const logoutFn = (): void => {
```

Comment:
can we use directly logout?

Reply:
Using logout directly cause the sonarlint error typescript:S6544
SonarLint: Promise-returning function provided to property where a void return was expected.

That's why I wrapped it in a non-promise-returning function